From 020e912087945a85ce4727d5a685e2cb8a364a51 Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Sat, 2 Nov 2013 01:36:34 +0800
Subject: [PATCH 258/602] ENGR00278179 add gpu memory tool into product
 package

this tool can be used to dump the gpu memory info

Signed-off-by: Xianzhong <b07117@freescale.com>
---
 imx6/imx6.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index d735818..6b8b5f2 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -158,6 +158,10 @@ PRODUCT_PACKAGES += \
 PRODUCT_PACKAGES += \
 	powerdebug
 
+# gpu debug tool
+PRODUCT_PACKAGES += \
+	gmem_info
+
 # Omx related libs, please align to device/fsl/proprietary/omx/fsl-omx.mk
 omx_libs := \
 	core_register					\
-- 
1.8.0

